src/components/Table/BaseTable.tsx

```diff
@@ -131,12 +132,13 @@ export const BaseTable = <T extends LGRowData>({
             `}
           >
             {row.getVisibleCells().map((cell) => (
-              <Cell key={cell.id}>
+              <Cell key={cell.id} style={{ padding: `${size.xxs} 2px` }}>
                 {flexRender(cell.column.columnDef.cell, cell.getContext())}
               </Cell>
             ))}
             {row.original.renderExpandedContent && (
-              <ExpandedContent row={row} />
+              // @ts-expect-error
+              <StyledExpandedContent row={row} />
```

Comment:
Why does turning this into a StyledComponent affect the typing for row?

Reply:
Could not figure out any way to handle this error 😢 
```
Type 'Row<LGTableDataType<T>>' is not assignable to type 'LeafyGreenTableRow<unknown>'.
  The types of 'original.renderExpandedContent' are incompatible between these types.
    Type '(row: LeafyGreenTableRow<T>) => Element' is not assignable to type '(row: LeafyGreenTableRow<unknown>) => Element'.
      Types of parameters 'row' and 'row' are incompatible.
```

Reply:
hmm I think it's worth flagging with the LG team because it looks like Emotion is having trouble inferring the generic types used by ExpandedContent. 

I managed to resolve this by adding a type assertion on the `StyledExpandedContent` like so
```ts
const StyledExpandedContent = styled(ExpandedContent)`
  // Allow expanded content containers to take up the full table width
  [data-state="entered"] > div {
    flex-grow: 1;
  }
` as typeof ExpandedContent;
```

Reply:
A Google search on this lead me to [this thread](https://github.com/emotion-js/emotion/issues/2342), it seems to be a known limitation on emotion and they also recommended the same approach.